### PR TITLE
Add simple moving average

### PR DESCRIPTION
**Pull request by philippegeraldeli**
# SMA with trade and candlestick options

### CODE REVIEW

File: tests/unit_test.rs [hunk 1]
```diff
@@ -598,3 +598,68 @@
 //         assert_eq!(extremes[6].retrace_ratio, Some(1.1282051282051282));
 //     }
 // }
+
+// #[cfg(test)]
```
Comment (aLexandreM1) [inline tests/unit_test.rs:602]: Should not be commited, right?

File: src/strategies/indicators/moving_average.rs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+fn calculate_sma(data: Vec<f64>, window: usize) -> Result<f64, MovingAverageError>{
+    if window == 0 {
+        return Err(MovingAverageError::ZeroWindowSize);
+    }
+    if data.len() < window {
+        return Err(MovingAverageError::WindowSizeTooLarge);
+    }
+
+    let mut average = 0.0;
```
Comment (aLexandreM1) [inline src/strategies/indicators/moving_average.rs:36]: Idk if I agree with `average` to be the sum of price value to divide later by window (window time)
its not average yet 🤔

**Review by aLexandreM1 — CHANGES_REQUESTED** [review]
Code looks very simple and its not a problem at all
But tests are commited and CI fails because there's no class used (dead code detected)
Lets use this case for our meeting and understand if we will recreate those indicators or we're going to breakdown and use built ones from the internet

**Comment by victorl2** [comment]
I think we should closed this PR, as we already discussed, to favor using the indicators implemented in a existing library

**Pull request closed without merging**